libs/model-utils/src/lib/utils/many-to-array.ts

```diff
@@ -0,0 +1,8 @@
+import { Many } from '../models';
+
+/**
+ * Convert a {@link Many} to an array. It gives an empty array if the given value is nullish.
+ */
+export function manyToArray<T>(x: Many<T>): T[] {
+  return x == null ? [] : Array.isArray(x) ? x : [x];
```

Comment:
can we use 
`!NotNil(x) ? [] : Array.isArray(x) ? x : [x];` 

or 

`NotNil(x) ? (Array.isArray(x) ? x : [x]) : [] `

or maybe not complicate much?

Reply:
`notNil` is really aimed for use in functional programming, like to be used in `Array::filter` or using `filter(notNil)` in an Observable pipe. This was done to not have to write things liek this all the time
```typescript
  const arr = [1, undefined, 2].filter(x => x != null) // not even sure that typescript sees this as number[], it might still see it as (number | undefined)[]

from([1, undefined, 2]).pipe(filter(x => x != null)).subscribe(console.log) // logs 1 & 2
```

But in cases like this in `manyToArray`, it's way easy to read to have `x == null` than to use `notNil` I would say 😸

Reply:
okay then :) good to know that :D